pkg/operator/controller/names.go

```diff
@@ -120,3 +134,74 @@ func NodeResolverDaemonSetPodSelector() *metav1.LabelSelector {
 		},
 	}
 }
+
+// CABundleCMVolAndVolMount takes a CA bundle ConfigMap name and a TLS server name, and returns
+// the ConfigMap Volume and VolumeMount to be used for the DaemonSet.
+func CABundleCMVolAndVolMount(caBundleName string, serverName string, caBundleRevisionMap map[string]string) (*corev1.Volume, *corev1.VolumeMount) {
```

Comment:
Oh sorry I should have been more clear - I didn't mean this whole `CABundleCMVolAndVolMount`. I think you had that in the right place, I don't think this function belongs here based on the pattern of other functions in names.go.

I was specifically referring to the string:
`"ca-bundle.crt"`

I noticed it was used here, and 3 other times. I was just wondering if it made sense to declare ca-bundle.crt as a const here?

Reply:
Ah totally misunderstood. Of course, it makes sense to declare it as const.